shared/oval/sshd_use_approved_ciphers.xml

```diff
@@ -4,12 +4,12 @@
       <title>Use Only Approved Ciphers</title>
       <affected family="unix">
         <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_debian</platform>
       </affected>
       <description>Limit the ciphers to those which are FIPS-approved and only
       use ciphers in counter (CTR) mode.</description>
     </metadata>
     <criteria operator="AND">
-      <extend_definition comment="Installed OS is certified" definition_ref="installed_OS_is_certified" />
```

Comment:
I am more inclined to the second approach mentioned by hex2a, with two XCCDF rules.

The requirements, or expectations, are different for certified and non-certified systems, so there is the need for two different rules that describes these differences in the algorithms.

At the OVAL level, the checks for certified OS and strong algorithms are independent and should be defined on their own, and the OVAL check for the two rules mentioned above can build up on that. One rule would just use the check of strong algorithms, while the other would check if strong algorithms are in use and they are certified.

So the "unit" being checked is combination of configuration and implementation of the algorithms.

What I don't like about the first approach is that the check of whether the system is certified and using strong algorithms will fall upon the combination of two rules. As long as the profiles are configure correctly things will work, but people customize and can easily unselect the check for `installed_OS_is_certified` and be fooled by the strong algorithm rule description mentioning FIPS.